README.md

```diff
@@ -1,24 +1,22 @@
-# NHS.UK prototyping kit
+# Urgent Care Signposting Prototypes
 
-This repository provides a development environment for making changes to components, along with research & findings, visual examples and code snippets from the Redesign project.
+Prototypes of the three possible solutions to improving urgent care signposting, conceived as a part of a Magnum CI design sprint.
+
+Created using <a href="https://github.com/nhsuk/frontend">NHS.UK prototyping kit</a>.
 
 Examples and code snippets of NHS.UK frontend components and elements can be found at our <a href="https://nhsuk-redesign.azurewebsites.net">NHS.UK prototyping and example website</a>.
 
 ## Installation
 
 Install the long-term support (LTS) version of <a href="https://nodejs.org/en/">Node.js</a>, which includes NPM.
 
-Clone the repo: `git clone https://github.com/nhsuk/frontend.git frontend` and whilst in the project directory `cd frontend`, install the required npm packages with: `npm install`
+Clone the repo: `git clone https://github.com/nhsuk/urgent-care-prototypes.git` and whilst in the project directory `cd urgent-care-prototypes`, install the required npm packages with: `npm install`
 
 Run the project in development mode `npm run watch` and visit <a href="http://localhost:3000">http://localhost:3000</a>.
 
 ## Contributing
 
-Follow our [contributing guide](https://github.com/nhsuk/frontend/blob/master/.github/CONTRIBUTING.md) to make changes to the NHS.UK frontend code base.
-
-## Deploying
-
-Following a successful pull request and merge, the code will automatically be deployed to our <a href="https://nhsuk-redesign.azurewebsites.net">NHS.UK prototyping and example website</a>.
```

Comment:
Would this be applicable for us also? Could add a placeholder for the URL in the meantime.